test/io/utils.dart

```diff
@@ -85,7 +85,7 @@ Future startServer() {
           requestBody = requestBodyBytes;
         }
 
-        var content = {
+        var content = <String, dynamic>{
```

Comment:
FYI, the reason for this change was strong mode inferred the map as `Map<String, Object>` and then complained about expression `content['headers'][name]` on line 98, saying that `Object` lacks `operator[]`.